modules/codegen/src/smithy4s/codegen/internals/Renderer.scala

```diff
@@ -1249,12 +1249,17 @@ private[internals] class Renderer(compilationUnit: CompilationUnit) { self =>
         renderPrismsEnum(name, values, hints, isOpen),
         values.map { case e @ EnumValue(value, intValue, _, hints) =>
           val valueName = NameRef(e.name)
-          val valueHints = line"$Hints_(${memberHints(e.hints)})"
+
+          val baseLine = line"""case object $valueName extends $name("$value", "${e.name}", $intValue, $Hints_())"""
```

Comment:
`$Hints_.empty` ?

Reply:
it does, I just wanted to avoid increasing the diff. Currently, this PR is a no-op for members that don't have hints.

We can do this change anytime without breakage, I suggest that we do it separately.